src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -44,10 +45,10 @@
      *            The id of this Backpack
      */
     public PlayerBackpack(@Nonnull PlayerProfile profile, int id) {
-        this(profile, id, profile.getConfig().getInt("backpacks." + id + ".size"));
+        this(profile, id, profile.getConfig().getInt( CONFIG_PREFIX+ id + ".size"));
 
         for (int i = 0; i < size; i++) {
-            inventory.setItem(i, cfg.getItem("backpacks." + id + ".contents." + i));
+            inventory.setItem(i, cfg.getItem( CONFIG_PREFIX+ id + ".contents." + i));
```

Comment:
Please place that space after the variable.
```suggestion
            inventory.setItem(i, cfg.getItem(CONFIG_PREFIX + id + ".contents." + i));
```

Reply:
64c639ef8